package.json

```diff
@@ -35,7 +35,25 @@
     "eject": "react-scripts eject"
   },
   "eslintConfig": {
```

Comment:
can this config sit in a different file? I'd almost prefer that, at least eventually